src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -49,6 +49,7 @@
 
         <service id="validator.email" class="Symfony\Component\Validator\Constraints\EmailValidator">
             <argument></argument>
+            <argument></argument>
```

Comment:
Would be good IMO to add comment that those two options are passed in extension.